### DISCUSSION TITLE
Migrating the server - any sufficiently in-depth documentation/guide on this?

### DISCUSSION THREAD

**Discussion by froghilda** [mbround18/valheim-docker#326] [Instructions]
As the title states, do I just suck at finding docs, or is there some documentation on this? My use case would involve deploying a lightsail resource (we want to try another region) to run the server that has an exact copy of the former's world ..before destroying the former lightsail resource. If there isn't sufficiently good docs, perhaps someone could outline the steps I would have to do to accomplish this. <3

Froggy

**Comment by chris948** [comment]
I can't really speak to lightsail, but when I moved my game files to a container on my NAS, I just set the name, password, world as identical to my old server and copied the two files that are in the world directory. *.db and *.fwl.

Took 2 minutes.

**Reply by mbround18** [reply]
I personally use croc to send world files when migrating. https://github.com/schollz/croc the docker image has all the requirements to be able to install it. I also use it to send BepInEx configs between my host and my desktop to share them out for ValheimPlus consistency. 

I can do a write up about it later :)

**Reply by mbround18** [reply]
@froghilda 

https://github.com/mbround18/valheim-docker/blob/mbround18/doc-nd-upgrade/docs/tutorials/how-to-transfer-files.md

**Comment by punksinatra** [comment]
Backup and copy the world files.. By lightsail I would assume amzon? In any case if you want the exact world just backup, copy to new host the world files
